Sources/IO/Misc/PDBReader/example/index.js

```diff
@@ -26,6 +26,7 @@ const sphereActor = vtkActor.newInstance();
 const stickActor = vtkActor.newInstance();
 
 filter.setInputConnection(reader.getOutputPort());
+filter.setHideElement('H');
```

Comment:
mmh well, yes I guess you right, but it seems working.
The code which check `model.hideElement` is: 
```
if (model.hideElement.includes(ATOMS[atomicNumber[i]].id)) {
  continue;
}
```
And it does work with all following settings
```
filter.setHideElement('H');
filter.setHideElement(['H']);
filter.setHideElement(['H', 'O']);
```